src/views/BrowseView.tsx

```diff
@@ -18,12 +18,19 @@ const BrowseView: React.FC<RouteComponentProps> = () => {
   const { loading: categoriesLoading, data: categoriesData, error: categoriesError } = useQuery<GetCategories>(
     GET_CATEGORIES,
     {
-      onCompleted: (data) => handleCategoryChange(data.categories[0]),
+      onCompleted: (data) => {
+        handleCategoryChange(data.categories[0])
+        firstCategoryChange.current = false
+      },
     }
   )
-
+  const headerRef = useRef<HTMLHeadingElement>(null)
+  const firstCategoryChange = useRef(true)
```

Comment:
1. Why do we need that?
2. Why can't it be a regular state?

Reply:
1. We want to scroll up whenever the category changes, but _only if_ it changed because a user selected another category. The `onCompleted` callback was calling `handleCategoryChange` and so the page was scrolling up down every time the Browse view was opened, regardless of user actions.
2. It can be state, changed it.

Reply:
Okay, so we need to skip the scroll on initial category load, got it. It's little confusing at the moment with the naming. Maybe we don't need state at all? We could add an optional param to the `handleCategoryChange` that would skip the scrolling

Reply:
Good point actually. I'll add the parameter there